### ISSUE TITLE
Running a test causes the report to re-open in the same window section

### ISSUE THREAD

**Issue by fwcd** [microsoft/vscode-java-test#727]
Previously, I could have the test report and the code side-by-side open while running tests:
![grafik](https://user-images.githubusercontent.com/30873659/59554891-76b07980-8faa-11e9-9527-0141f0bd3e11.png)

However, if I click `Run Test` in the current version of the test runner, the report automatically re-opens in the "code window section":

![grafik](https://user-images.githubusercontent.com/30873659/59554910-c8590400-8faa-11e9-9356-4f772863717c.png)

It would be useful if the test report could stay in the same section instead, as it was previously the case.

Test runner version: 0.18.0
VSCode version: 1.35.0
OS: Windows_NT x64 10.0.17134

**Comment by jdneo** [comment]
Hi @fwcd,

What's the value of your `java.test.report.position` setting? Do you set it to `currentView`?

If that is the case, you can set it to `sideView` to make the report always show in the second column.

![image](https://user-images.githubusercontent.com/6193897/59574263-24c23d80-90e9-11e9-99b4-324109b7a50a.png)

**Comment by fwcd** [comment]
Hi, I did not change the default value, which seems to be `sideView`:

![grafik](https://user-images.githubusercontent.com/30873659/59604462-8e077800-910c-11e9-9f45-9c53b1af1c7f.png)

It seems as if the test report always chooses the second window column regardless of where the code is located in which I started the test. This leads to strange issues where, e.g. the following layout

```
+--------+---------+
| report |  code   |
+--------+---------+
```

switches to

```
+--------+---------+
|     report       |
+--------+---------+
```

since the report tries to move into the second column thus leaving the first column empty and making it disappear.

**Comment by jdneo** [comment]
I see...

Honestly, things might become tricky in such a scenario. What I think is a safe fix is that: 
- When the report is not opened in the editor, reveal it as the setting specified.
- When the report is opened in the editor, just update the report and do not change its column.

@fwcd what do you think?

**Comment by fwcd** [comment]
That sounds good!

**Comment by jdneo** [comment]
Fixed in #737. The fix will be released in `0.18.1`
